### ISSUE TITLE
Replace TYGS, or introduce it earlier?

### ISSUE THREAD

**Issue by widdowquinn** [sipbs-compbiol/BM329_Block_A_workshop#6]
TYGS is _the_ go-to classification server, but its turnaround time - especially under load - is a bit slow for the workshop.

The current placement in the workshop is late on because we build concepts/increase precision from 16S through MLST to whole-genome analysis. @mafeeney suggested presenting the TYGS section earlier, which I didn't want to do for the introduction of concepts but - if we implement #1 then we can keep TYGS at the end of the introductory section, and encourage students to do the TYGS analysis for the _Brucella_/_Ochrobactrum_ case early on in that analysis - giving them chance of getting the TYGS result back quicker.

An option would be to replace TYGS, but I think the link to LPSN/DSMZ is important, as is seeing the real tool in action.

**Comment by mafeeney** [comment]
To clarify, I did not mean to present the TGYS section earlier - I meant something along the lines of adding a first step, do XYZ because the jobs will take a long time to run - then by the time the students have worked through the rest of the material, the job should hopefully have run and they can look at/interpret their results then, in the same sequence as this year. The order of events works very well as is, I would not change that.

**Comment by widdowquinn** [comment]
I see - I was concerned about asking them to run TYGS on the genome sequence without a proper explanation _before_ doing the 16S classification.

On one level it's a bit like "put the water on to boil now, I'll tell you why, later, and you'll thank me" but, in another sense we're actually _doing_ the analysis by submitting the sequence rather than just _preparing for_ a later analysis.

The practicality might outweigh my preference for a strict order of concepts, though ;)

**Comment by mafeeney** [comment]
Yes, I agree that ideally the concepts should be presented in the current order. 

However, the "boiling water now" analogy is apt and fits in well with what we tell them in the prac labs (time management skills, starting experiments that require long incubation times first...)
